libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/AdaptiveDialog.cs

```diff
@@ -406,6 +464,74 @@ protected async Task<bool> ProcessEventAsync(SequenceContext sequenceContext, Di
             return handled;
         }
 
+        protected async Task<bool> ProcessFormAsync(SequenceContext sequenceContext, CancellationToken cancellationToken)
```

Comment:
>ProcessFormAsync [](start = 35, length = 16)

Is this processForm?  Is there a better name for this?  Is it process entities?  or SlotMapping? or ???

Reply:
Renamed to ProcessQueuesAsync.

---
In reply to: [360164036](https://github.com/microsoft/botbuilder-dotnet/pull/2936#discussion_r360164036) [](ancestors = 360164036)